Simonyi/Non-Standard-Operations/MTCS/MTPtg/MTPtg-Configuration-for-MTRot-MTMount.rst

```diff
@@ -0,0 +1,194 @@
+
+             data:
+                 -
+                   - MTPtg 
+                   - STANDBY
+..
+
+
+2. Find the name of the **pod** where the ``MTPtg`` is running. 
+   
+    From your terminal, run the following command which list all the **pods** related to the *<namespace>* maintel:
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml get pod -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName -n maintel
+
+    ..
+
+    .. figure:: ./_static/1.png
+      :width: 950px
+      :height: 165px
+      :name: Your figure
+
+      In this particular case the name of the ``MTPtg`` **pod** is **mtptg-djhpv**.
+    ..  
+
+3. Connect to the ``MTPtg`` **pod mtptg-djhjv** within the *<namespace>* maintel. 
+
+    The command will open a terminal within the pod.
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml exec --stdin --tty mtptg-djhpv -n maintel -- /bin/bash
+
+    ..
+   
+    .. figure:: ./_static/2.png
+        :width: 900px
+        :height: 65px
+
+    ..
+
+4. **Move to configuration directory,** the configuration files are one level up. The directory contains the configuration files :file:`MTPtg.info` (:file:`ATPtg.info` for AuxTel) and the pointing models :file:`mt.mod` files (:file:`at.mod` for AuxTel). 
+
+    At startup, the pointing component loads by default the pointing model :file:`mt.mod` file and the :file:`MTPtg.info` (equivalent for AuxTel).
+
+    .. prompt:: bash
+
+     [saluser@podname] cd /home/saluser/repos/ts_pointing_common/install/data
+
+    ..
+    
+    .. figure:: ./_static/3.png
+       :width: 900px
+       :height: 420px
+        
+    ..
+
+5. To edit the :file:`MTPtg.info` file use a text editor such as *vi*. 
+
+    5.1. ``MTRotator``: The **disable_rotator** line of the :file:`MTPtg.info` file contains the parameter you need to change. It reflects whether the ``MTRotator`` is monitored in the tracking.
+    
+    * The parameter *set* is **1**, means that it's being ignored and will not be commanded by the ``MTPtg`` component.
+
+    .. code-block::
+        :caption: MTPtg.info: disable_rotator: 1
+
+            - Set 0 → enabled
+            - Set 1 → disabled
+    ..
+
+
+    5.2. ``MTMount``: The line containing the **disable_mount** parameter in the :file:`MTPtg.info` file is the one to edit. It shows whether the ``MTMount`` is monitored in the tracking.
+   
+    * The parameter *set* is *0*, which means that it's being included and commanded by the ``MTPtg`` component.
+   
+    .. code-block::
+        :caption: MTPtg.info: disable_mount: 0
+
+            - Set 0 → enabled
+            - Set 1 → disabled
+..
+
+
+6. **Exit** the **pod** by typing :command:`exit`.
+
+7. From LOVE, **send** the ``MTPtg`` back to ``ENABLED`` using the script :file:`set_summary_state.py` and configuration below. ``MTMount`` must be ``ENABLED``, even if not tracking, so ``CCW`` can be still monitored.
```

Comment:
I suggest rewriting as 
'Send the MTPtg to ENABLED using the script set_summary_state.py and configuration below in the MTQueue.
Note: MTMount must be ENABLED, even if not tracking, so CCW can be still monitored.
'